Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -628,6 +629,23 @@ private static void RemoveTempFileFromUserDrive(IEnvironment environment)
             FileSystemHelpers.DeleteDirectorySafe(Path.Combine(environment.RootPath, "data", "Temp"), ignoreErrors: true);
         }
 
+        // add/update gitconfig file to be equivalent of running
+        // git config --global --add safe.directory "*"
+        private static void EnsureGitConfigFileWithSafeDirectoryConfig(IEnvironment environment)
```

Comment:
for safe rollout, I would hide this under ScmConfigurations (disabled by default).

Reply:
updated, thanks!